engines/bops_api/spec/services/application/search_service_spec.rb

```diff
@@ -172,6 +172,66 @@
       end
     end
 
+    context "when filtering by application status" do
+      let(:local_authority) { create(:local_authority) }
+
+      let!(:app1) { create(:planning_application, :in_assessment) }
+      let!(:app2) { create(:planning_application, :to_be_reviewed) }
+
+      context "when no application type codes are provided" do
+        let(:params) { {applicationStatus: [""]} }
+
+        it "returns matching applications" do
```

Comment:
Bit picky but you could change the assertion to "returns no applications" (though technically I guess that is still matching applications)